### ISSUE TITLE
Question: Masked Loss for LukeForEntitySpanClassification

### ISSUE THREAD

**Issue by zhenbangt** [huggingface/transformers#12047]
In transformers.LukeForEntitySpanClassification, the loss is calculated from labels of shape `(batch_size, entity_length)` and in range `[0, ..., config.num_labels - 1]`. I did not see in the source code you masked out the loss for padded tokens, nor did you make a special label for padded tokens. So how do you deal with the loss of padded sequences?

Many thanks!

**Comment by NielsRogge** [comment]
Normally, in PyTorch, you have to set labels for padding tokens equal to -100, as -100 is the default ignore index that loss functions use.
